cas/incident-report.md

```diff
@@ -1,169 +1,384 @@
+
+- Providing additional information,
+- Asking clarifying questions,
+- Discussing technical aspects of the incident,
+- Suggesting corrective actions,
+- Highlighting opportunities for improvement,
+- Contributing lints to open source linting projects to promote improved future issue detection/prevention, and
+- Sharing lessons learned from past experiences.
+
+Individuals representing CA Owners are especially encouraged to participate broadly in the Incident Reporting process, extending their contributions beyond incidents involving only their own organization. Sharing insights and perspectives across organizational boundaries fosters a collaborative learning environment and strengthens the overall security posture of the Web PKI ecosystem.
+
+To ensure productive dialogue, please keep all comments constructive, relevant to the report, and in line with the [CCADB Code of Conduct](https://docs.google.com/document/d/19ALqEvHtTE6OUTz2FaOXrU9gruIdvia5EDh3hXeGpZA/edit#heading=h.cumc0pgd1s7c).
+
+### How do I submit an Incident Report?
+Create a new Bugzilla issue by filling out the Summary and Description fields of [this form](https://bugzilla.mozilla.org/enter_bug.cgi?format=__default__&product=CA%20Program&component=CA%20Certificate%20Compliance&bug_type=task). 
+
+The "Summary" field in Bugzilla (i.e., "Subject line") MUST begin with the CA Owner’s name, followed by a colon, and a brief title that highlights the type of incident being reported (e.g., "EXAMPLE CA OWNER: Incorrect Subject RDN Encoding"). 
+
+The "Description" field MAY contain a Preliminary or Full Incident Report (copied and pasted from the corresponding markdown template, as explained [below](#incident-report-templates)). 
+
+### What format should be used?
+Use one of the templates below, depending on the type of Incident Report being disclosed:
+
+- [Preliminary Incident Report](#preliminary-incident-report) (for third party reporters and CA Owners)
+- [Full Incident Report](#full-incident-report) (for CA Owners)
 
-The purpose of incident reporting is to help us work together to build a more secure web. Therefore, the incident report should share lessons learned that could be helpful to all CA Owners in building better systems. The incident report should explain how systems or processes failed, how the mis-issuance or incident was made possible, and why the problem was not detected earlier. In addition to the timeline of responding to and resolving the incident, the incident report should explain how the CA Owner's systems or processes will be made more robust, and how others may learn from the incident.
+If being reported by the CA Owner corresponding with an incident, all fields included in the relevant template MUST be completed.
```

Comment:
It might be helpful to add that fields that are not applicable (e.g. Appendix) shall be included in the report and marked as such. I'm not a big fan of being so prescriptive, but I think the "MUST be completed." language takes us down this road.